internal/config/cgmgr/cgroupfs.go

```diff
@@ -0,0 +1,83 @@
+// +build linux
+
+package cgmgr
+
+import (
+	"fmt"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/containers/libpod/pkg/cgroups"
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+type CgroupfsManager struct{}
+
+const (
+	defaultCgroupfsParent = "/crio"
+)
+
+// Name returns the name of the cgroup manager (cgroupfs)
+func (*CgroupfsManager) Name() string {
+	return cgroupfsCgroupManager
+}
+
+// IsSystemd returns that this is not a systemd cgroup manager
+func (*CgroupfsManager) IsSystemd() bool {
+	return false
+}
+
+// ContainerCgroupPath takes arguments sandbox parent cgroup and container ID and returns
+// the cgroup path for that containerID. If parentCgroup is empty, it
+// uses the default parent /crio
+func (*CgroupfsManager) ContainerCgroupPath(sbParent, containerID string) string {
+	parent := defaultCgroupfsParent
+	if sbParent != "" {
+		parent = sbParent
+	}
+	return filepath.Join(parent, scopePrefix+"-"+containerID)
```

Comment:
Why is there a scopePrefix for cgroup fs? If we do want a prefix at all, we should call it something else, so it doesn't get mixed up with systemd scopes.

Reply:
updated to `crioPrefix` (it's just "crio")